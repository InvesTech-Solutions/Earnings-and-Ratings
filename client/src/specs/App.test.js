import React from 'react';
import { shallow, mount, render } from 'enzyme';
import App from './app.js';

describe('component: App', () => {
  it('should render', () => {
    const wrapper = shallow(<App />);
    const exampleLoaded = 1;
    console.log(exampleLoaded);
    expect(exampleLoaded).toEqual(1)
  });
});
