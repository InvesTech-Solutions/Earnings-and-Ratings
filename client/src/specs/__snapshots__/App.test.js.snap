// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component: App should render initial layout 1`] = `
Array [
  <div>
    <p>
      yo
    </p>
  </div>,
]
`;
